client/src/containers/SingleArticle/SingleArticle.js

```diff
@@ -21,9 +21,10 @@ class SingleArticle extends Component {
       })
   }
   showArticleImage() {
+    const imageUrl = process.env.REACT_APP_API_URI || 'http://localhost:3001/images/';
```

Comment:
`/images` should be in the `<img src` part - otherwise it will not be added to REACT_APP_API_URI

Reply:
I do not understand what do you mean?

Reply:
ok I have got it